Dockerfile

```diff
@@ -1,21 +1,24 @@
 FROM alpine:3.19 AS base_gcc
```

Comment:
End of support of Alpine Linux v3.19.x is on Nov 1 this year. We should migrate to 3.20 or 3.21 later.